### PR TITLE
Setup

### PR DESCRIPTION
**Pull request by bun919tw**
Most of the code structure is copied from plasma_mvp.

Install requirements:
`pip install -r requirements.txt`

Ganache-cli command:
`ganache-cli -m =plasma_cash`

Run child chain:
`FLASK_APP=plasma_cash/child_chain/server.py flask run --port=8546`

Deposit:
```
python
>>> from plasma_cash.client.client import Client
>>> c = Client()
>>> c.deposit(100, '0xb83e232458a092696be9717045d9a605fb0fec2b')
```

### CODE REVIEW

File: requirements.txt [hunk 1]
```diff
@@ -0,0 +1,6 @@
+py-solc
```
Comment (boolafish) [inline requirements.txt]: better to have version as well

File: plasma_cash/client/child_chain_service.py [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+            response = requests.request(
+                           method=method,
+                           url=url,
+                           params=params,
+                           data=data,
+                           headers=headers,
+                           verify=self.verify,
+                           timeout=self.timeout,
+                       )
+        except requests.exceptions.Timeout:
```
Comment (boolafish) [inline plasma_cash/client/child_chain_service.py]: why not just let it throws the exception it self?

File: plasma_cash/client/child_chain_service.py [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+            raise Exception
+        except requests.exceptions.ConnectionError:
+            raise Exception
+
+        if response.ok:
+            return response
+        else:
+            raise Exception
```
Comment (boolafish) [inline plasma_cash/client/child_chain_service.py]: would recommend to throw custom exception and put the response (message?) into Exception message to show what's happening here.

File: plasma_cash/client/client.py [hunk 4]
```diff
@@ -0,0 +1,14 @@
+from plasma_cash.root_chain.deployer import Deployer
+from .child_chain_service import ChildChainService
+
+
+class Client(object):
+
+    def __init__(self,
+                 root_chain=Deployer().get_contract('RootChain/RootChain.sol'),
+                 child_chain=ChildChainService('http://localhost:8546')):
```
Comment (boolafish) [inline plasma_cash/client/client.py]: recommend the `localhost:8546` to be in config though. People might want to change in different env?

**Comment by kasima** [comment]
This looks like a great start. Looks good to merge from me :+1:

**Comment by DavidKnott** [comment]
Great work!

**Pull request merged**